agent/cache/cache.go

```diff
@@ -37,6 +37,10 @@ import (
 var Gauges = []prometheus.GaugeDefinition{
 	{
 		Name: []string{"consul", "cache", "entries_count"},
+		Help: "Deprecated - please use cache_entries_count instead.",
```

Comment:
Nice! I think ideally, this is all that is needed.

I think, in practice, IIRC, these messages only get "emitted" on the dummy metrics like https://github.com/hashicorp/consul/issues/13495 . Apologies if you are already well aware of this but I think for real world metrics (especially label based), the help message is not populate properly. But that's not something to be fixed in `consul`, rather something to get fixed in `go-metrics` I think.

Reply:
Yep. That will have to be fixed separately.

Reply:
Should we get this fixed in `go-metrics` first so that we can deliver the help message?

Reply:
there is a workaround for getting the help strings - as acpana pointed to in #13495 - and I think we should consider that a separate issue with bigger scope